src/App.tsx

```diff
@@ -1,26 +1,239 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useRef, useState } from 'react';
+import {
+  addTodo,
+  deleteTodo,
+  getTodos,
+  updateTodo,
+  USER_ID,
+} from './api/todos';
+import { Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import classNames from 'classnames';
+import { AddTodo } from './components/AddTodo/AddTodo';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { Filter } from './types/Filter';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState<Filter>(Filter.all);
+  const [newTodo, setNewTodo] = useState(''); //input element
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [loader, setLoader] = useState<number>(0);
+  const [isSubmiting, setIsSubmiting] = useState(false);
+  const inputRef = useRef<HTMLInputElement>(null); //focus on input element
+
+  const handleErrorMessage = (message: string) => {
+    setErrorMessage(message);
+    setTimeout(() => {
+      setErrorMessage('');
+    }, 3000);
+  };
+
+  useEffect(() => {
+    inputRef.current?.focus();
+
+    getTodos()
+      .then(response => {
+        setTodos(response);
+        setErrorMessage('');
+      })
+      .catch(() => {
+        handleErrorMessage('Unable to load todos');
+      });
+  }, []);
+
+  function getFilterTodos(todoForFilter: Todo[], enumFilter: number): Todo[] {
```

Comment:
It is important to stick to one way of creating functions, fix it everywhere 
```suggestion
  const getFilterTodos = (todoForFilter: Todo[], enumFilter: number): Todo[] => {
```